content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,374 @@
+- _beneficiary_: gain of the _offending_ validator's protocol rewards as the _reporting_ validator of a slashed _fault_.
+
+Autonity community play the following economic role in AFD processing:
+
+- _beneficiary_: gain of slashed stake token for community funds.
+
+### Protocol primitives
+
+Essential primitives of AFD are: accusation, innocence, and fault proofs; slashing and severity; jailing.
+
+#### Accusations
+
+An _accusation_ is a claim that a consensus committee member has failed to participate correctly in consensus. I.e. it has failed to adhere to a rule governing validator participation in consensus and so is guilty of a rule infraction. Protocol only allows a validator to be under _accusation_ once at a time. New _accusations_ are made by submitting evidence of an accusable rule infraction on-chain as an accountability event of type `Accusation`.
+
+Accusations do not automatically cause slashing and an _innocence proof window_ measured in blocks gives the accused _offending_ validator a window to detect an _accusation_ and prove _innocence_ by submitting an _innocence_ proof on-chain.
+
+There are protocol constraints on when an _accusation_ can be made. A _reporting_ validator can only submit an _accusation_ _proof_ if:
+
+- the _offending_ validator:
+  - does not already have a _fault_ in the epoch for which the _new accusation_ is being made for an offence with a higher _severity_
+  - is not currently already accused of committing a rule infraction, i.e. there is a _pending accusation_. In this case, a _new accusation_ cannot be made until expiry of the _innocence proof window_. This creates a _deadline_ measured in block height before which a new `accusation` cannot be submitted.
+- it is within the _accusation window_:
+  - an accusation must be made `<= 256` blocks after the detected accountability event.
+
+As each block is finalised, AFD will attempt to promote _accusations_ where the _innocence proof submission window_ has expired within protocol constraints to proven _faults_.
+<!--
+To check if an _offending_ validator has a _pending accusation_, a _reporting_ validator calls protocol functions:
+
+- [`canAccuse()`](/reference/api/accountability/#canaccuse): to determine if (a) an offending validator is accusable, and, (b) the _deadline_ number of blocks remaining in the _innocence proof submission window_ if there is a _pending accusation_.
+- [`canSlash()`](/reference/api/accountability/#canslash): to determine if the  _offending_ validator already has a _fault_ for a rule infraction in the epoch with a _severity_ higher than the new accusable rule infraction detected.
+-->
+After successful [handling and verification](/reference/api/aut/op-prot/#handleevent-accountability-contract) of an _accusation_ on-chain a `NewAccusation` event is emitted logging the _offending_ [validator identifier](/concepts/validator/#validator-identifier) address, _severity_ of rule infraction, and the event ID.
```

Comment:
I would move this after the 2nd paragraph.

Reading it after the paragraph that explains accusation promotion gives the idea that the `NewAccusation` is issued when the accusation is promoted.

Reply:
Makes sense. Moved in commit 2f02b70a085047615a816682b56ed4fbd10cee6c

Reply:
Makes sense @lorenzo-dev1 - moved in commit 2f02b70a085047615a816682b56ed4fbd10cee6c.